### PR TITLE
Native token treasury graph

### PR DESCRIPTION
**Pull request by NoahSaso**
Add graph of treasury history because graphs are awesome! Also added pagination and made treasury history lines cleaner (with arrows instead of words).

Screen Shot 2022-08-15 at 3 15 16 PM

Screen Shot 2022-08-15 at 3 14 25 PM

### CODE REVIEW

**Review by elsehow — APPROVED** [review]
This looks great!

**Comment by onewhiskeypls** [comment]
checked these changes against mainnet neta dao (juno1c5v6jkmre5xa9vf9aas6yxewc7aqmjy0rlkkyk4d88pnwuhclyhsrhhns6) and this happened:

image

File: apps/dapp/components/dao/DaoTreasuryHistory.tsx [hunk 1]
```diff
@@ -43,117 +58,208 @@ export const DaoTreasuryHistory = (props: DaoTreasuryHistoryProps) => {
...
+        } finally {
+          setLoading(false)
+        }
+      },
+    [
+      coreAddress,
+      setLoading,
+      setTransactions,
+      setLowestHeightLoaded,
+      setLowestHeightLoadedTimestamp,
```
Comment (ebaker) [inline apps/dapp/components/dao/DaoTreasuryHistory.tsx:141]: might wanna add `BLOCK_HEIGHT_INTERVAL` to this array, but should be fine without

**Review by ebaker — APPROVED** [review]
nice work!

lgtm :+1:

**Comment by NoahSaso** [comment]
> 

Resolved in https://github.com/DA0-DA0/dao-dao-ui/pull/811/commits/0a83f6a1dbd5dc2dc37377a59ab3d62942693cef :D

**Comment by NoahSaso** [comment]
> checked these changes against mainnet neta dao (juno1c5v6jkmre5xa9vf9aas6yxewc7aqmjy0rlkkyk4d88pnwuhclyhsrhhns6) and this happened:
> 
> image

Resolved in https://github.com/DA0-DA0/dao-dao-ui/pull/811/commits/0a83f6a1dbd5dc2dc37377a59ab3d62942693cef :D

**Pull request merged**